cmd/entrypoints/clusterresource.go

```diff
@@ -41,12 +48,37 @@ func getClusterResourceController(ctx context.Context, scope promutils.Scope, co
 		panic(err)
 	}
 
-	clientSet, err := admin.ClientSetBuilder().WithConfig(admin.GetConfig(ctx)).Build(ctx)
-	if err != nil {
-		panic(err)
+	var adminDataProvider interfaces.FlyteAdminDataProvider
+	if configuration.ClusterResourceConfiguration().IsStandaloneDeployment() {
+		clientSet, err := admin.ClientSetBuilder().WithConfig(admin.GetConfig(ctx)).Build(ctx)
```

Comment:
What happens when admin has auth enabled.
Does this need to be built with tokenCacheProvider?

Reply:
it shouldn't, it's never called by an end user's machine but from a service